src/components/Results/use/DownloadForm.ts

```diff
@@ -0,0 +1,130 @@
+import { ResultsPerQuestion, isArrayQuestion } from './statistics'
+import { getTimeLimit } from '/@/components/UI/use/useOptions'
+
+type FormInfo = {
+  header: SpreadForm
+  rows: SpreadForm[]
+}
+type SpreadForm = {
+  readonly traq_id: string
+  readonly response_id?: number
+  readonly submitted_at: string
```

Comment:
jsでだけ扱うkey名はcamelCaseにしてください
```suggestion
  readonly traqID: string
  readonly responseID?: number
  readonly submittedAt: string
```